composables/drop/massmint/useDropMassMint.ts

```diff
@@ -0,0 +1,233 @@
+import {
+  AllocatedNFT,
+  DoResult,
+  allocateCollection,
+  allocateClaim as claimAllocation,
+} from '@/services/fxart'
+import useGenerativePreview, {
+  EntropyRange,
+  GenerativePreviewItem,
+} from '../useGenerativePreview'
+import { ImageDataPayload } from '../useGenerativeIframeData'
+import { ToMintNft } from '@/components/collection/drop/types'
+import { getFakeEmail } from '@/components/collection/drop/utils'
+import useDropMassMintPreview from './useDropMassMintPreview'
+import useDropMassMintUploader from './useDropMassMintUploader'
+import { useCollectionEntity } from '../useGenerativeDropMint'
+
```

Comment:
just general feeling looking at this code

is that the functions are complicated

and some operations should be extracted to smaller single resposiblity function

to improve readability

Reply:
for the sake of time I wont be making any major changes to the code

please feel free to create a follow up 

will gladly take it

Reply:
fair enough